packages/bpk-mixins/src/mixins/_buttons.scss

```diff
@@ -152,17 +152,28 @@
 ///   }
 
 @mixin bpk-button--destructive {
-  color: $bpk-button-destructive-color;
-  box-shadow: $bpk-button-destructive-box-shadow;
+  @include bpk-themeable-property(color, --bpk-button-destructive-text-color, $bpk-button-destructive-color);
+  @include bpk-themeable-property(background-color, --bpk-button-destructive-background-color, $bpk-button-destructive-background-color);
+  @include bpk-border-lg($bpk-button-destructive-border-color);
+  @include bpk-border-lg(var(--bpk-button-destructive-border-color, $bpk-button-destructive-border-color));
 
   @include bpk-hover {
-    color: $bpk-button-destructive-hover-color;
+    background-image: $bpk-button-destructive-hover-background-image;
```

Comment:
Why was this introduced?

Reply:
It was taken from the secondary bpk-button, so assumed this was needed to be the same

Reply:
Looks like it makes no difference so we can remove it